packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -8,7 +8,7 @@ import { HandledPromise } from '@agoric/eventual-send';
  * @typedef {import('../zoe').Invite} Invite
  * @typedef {import('../zoe').OfferHook} OfferHook
  * @typedef {import('../zoe').CustomProperties} CustomProperties
- * @typedef {any} TODO Needs to be typed
```

Comment:
I don't know the answer to this question, but do we need to retain this line? It looks like nothing changed in the file to change this line.